app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -93,6 +125,70 @@ public KSCurrency(final @NonNull CurrentConfigType currentConfig) {
     }
   }
 
+  /** Show's the project in the user's preferred currency. If the user has no preferred currency the project is shown
+   * in $ as a default if the user is in the US. If the user is located outside of the US the default will show as
+   * $US.
+   */
+  private @NonNull CurrencyOptions userCurrencyOptions(final float value, final @NonNull Project project, final String symbol) {
+    final Float fxRate = project.fxRate();
+
+    return CurrencyOptions.builder()
+        .country(project.country())
+        .currencyCode("")
```

Comment:
Do we need this?

Reply:
We need this because it's defined in the CurrencyOptions Model class. If we remove it, it won't build properly but also there's many other classes that use this KSCurrency class removing it altogether would affect the other classes.

Reply:
I mean do we need it in this method?

Reply:
Yes because we will get this error if it's removed:
`Caused by: java.lang.IllegalStateException: Missing required properties: currencyCode`

I think later we can actually remove it once we scope out how currency should be handled in those other classes.